php-iban.php

```diff
@@ -1084,6 +1084,25 @@ function _iban_nationalchecksum_implementation_si($iban,$mode) {
  return _iban_nationalchecksum_implementation_mod97_10($iban,$mode);
 }
 
+# Implement the national checksum for Slovak Republic (SK) IBAN
+# Source of algorithm: https://www.nbs.sk/_img/Documents/_Legislativa/_Vestnik/OPAT8-09.pdf
+# Account number is currently verified only, it's possible to also add validation for bank code and account number prefix
+function _iban_nationalchecksum_implementation_sk($iban,$mode) {
+ if ($mode !== 'verify') {
+  return '';
+ }
+
+ $account = iban_get_account_part($iban);
+ $weights = [6, 3, 7, 9, 10, 5, 8, 4, 2, 1];
```

Comment:
This needs to be changed to `array(...)` for backward compatibility with PHP5.2/5.3 in order to pass tests.